starter-code/spec/VikingSpec.js

```diff
@@ -8,50 +8,50 @@ describe("Soldier", function () {
   });
```

Comment:
Nos gusta como lucháis en cada ejercicio. Se nota el esfuerzo y seguid así y los resultado llegarán solos. Vamos!!!